src/api/journalEntries/journalEntries.ts

```diff
@@ -2,7 +2,7 @@ import API from '../apolloClient';
 import { gql } from '@apollo/client';
 import { useQuery } from 'react-query';
 
-const journalEntries = gql`
+export const journalEntries = gql`
```

Comment:
Used in journalEntry.ts to retrieve the user's journal entries before updating the cache for the display on the dashboard

Reply:
Could probably break out the queryFn instead